source/js/feature-speaker.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-plusplus */
+const featureSpeaker = document.getElementById('feature-speaker');
+const data = [
+  {
+    image: 'source/images/jay-cutler.jpg',
+    cardHeader: 'Jay Cutler',
+    job: 'Professional Body Builder.',
+    cardText: 'Gread place for trining. Great job on creating this Gym.',
+  },
+  {
+    image: 'source/images/andri.jpg',
+    job: 'Professional Body Builder.',
+    cardHeader: 'Andri',
+    cardText: 'WOW! this is one of the best place i have trine great equipment love it.',
+  },
+  {
+    image: 'source/images/body.jpg',
+    job: 'Professional Body Builder.',
+    cardHeader: 'Ali Mohammad',
+    cardText: 'Very clean and well equipment. Love the instactore for working very hard on us.',
+  },
+  {
+    image: 'source/images/women1.jpg',
+    job: 'Professional Body Builder.',
+    cardHeader: 'Anila Fergosen',
+    cardText: 'Very clean and gread place to train your body.',
+  },
+  {
+    image: 'source/images/women2.jpg',
+    job: 'Professional Body Builder.',
+    cardHeader: 'Ana Lee',
+    cardText: 'Best place for perpearing for Professional carrear.',
+  },
+  {
+    image: 'source/images/phile-health.jpg',
+    job: 'Professional Body Builder.',
+    cardHeader: 'jo Lee',
+    cardText: 'Best place for perpearing for Professional carrear.',
+  },
+];
+for (let x = 0; x < data.length; x++) {
+  const featureSpeakerBody = `<div class="feature-card">
+  <!-- card -image -->
+  <div class="feature-card-image">
+    <img src="${data[x].image}" alt="${data[x].cardHeader}">
+  </div>
+  <!-- card description (Body) -->
+  <div class="feature-card-body">
+    <h6>${data[x].cardHeader} <span id="feature-card-job">${data[x].job}</span></h6>
+    <p>${data[x].cardText}</p>
+  </div>
+</div>`;
+  // add to html
+  featureSpeaker.insertAdjacentHTML('beforeend', featureSpeakerBody);
+}
```

Comment:
### Your Speaker Card
![Screenshot from 2022-09-15 17-17-14 (copy)](https://user-images.githubusercontent.com/93039359/190402928-a45d8b06-22d9-48d5-ae22-d5b9ab7a6c33.png)

### Behance Speaker Card
![Screenshot from 2022-09-15 17-18-47 (another copy)](https://user-images.githubusercontent.com/93039359/190402947-891a40ee-507a-471e-9f53-6fff0a2fd7a3.png)

- [ ] You are missing the black n white box background in each card of speaker. Please add it to make your desing look professinal and similar to the Behance's design as requested by the previous reviewer :blush: